Player Movement

Belize22 opened the pull request:
- Player can move left and right.
- Player is able to jump and free-falls to the ground while in midair. Jumping can only be done while on the ground.

Belize22 committed:
Setup bindings for player, added basic movement.

- Player object created and key bindings have been set up.
- Player can walk left and right.
- Player can jump (must add condition where player can't jump
  in midair).

Belize22 committed:
Player jump works

- Jump now only occurs when the player is grounded.
- Serialized fields added (to allow tweaking of jump and movement
  speeds).

Megan-VanHumbeck commented on Assets/CodeBase/Entities/player/player.cs:
```diff
@@ -4,15 +4,76 @@
 
 public class player : MonoBehaviour
```
Class names/File names should be capitalized in C#

Belize22 committed:
Refactoring

- Classes were modified to be capitalized (C# convention).
- Some scene and asset data were saved.

Megan-VanHumbeck commented:
The files are no longer loadable in unity because the file name does not match the class name. This isn't necessary for C# (though it is best practice) but it is necessary for Unity. 

`The associated script can not be loaded. Please fix any compile errors and assign a valid script`

Belize22 committed:
Workaround commit for file rename

- Case sensitive exclusive changes does not seem to be detected by unity.

Belize22 committed:
Replaced translations with Rigidbody forces.

- Needs to be tweaked (feels similar to ice physics as of now).

Belize22 committed:
Finalized changes.

- Files are now changed to proper casing and have the relevant
content.

JesseDes commented on Assets/CodeBase/Entities/player/Player.cs:
```diff
@@ -0,0 +1,81 @@
+
+    private void OnCollisionEnter2D(Collision2D collision)
+    {
+        Debug.Log("Fuck");
```
Dont forget to clean logs we dont want the grader to see these LOL

DavidoBod replied:
hhahaha

Belize22 committed:
Merge branch 'master' into player-movement

Belize22 committed:
Resolved merge conflict.

- All related movement now uses RigidBody2D forces.
